pkgs/development/python-modules/dnspython/default.nix

```diff
@@ -2,26 +2,40 @@
 , buildPythonPackage
 , fetchPypi
 , pythonOlder
+, setuptools-scm
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "dnspython";
-  version = "2.1.0";
+  version = "2.2.0";
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    extension = "zip";
-    sha256 = "e4a87f0b573201a0f3727fa18a516b055fd1107e0e5477cded4a2de497df1dd4";
+    extension = "tar.gz";
+    sha256 = "1mi6l2n766y1gic3x1swp2jk2nr7wbkb191qinwhddnh6bh534z7";
   };
 
-  # needs networking for some tests
-  doCheck = false;
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # dns.exception.SyntaxError: protocol not found
```

Comment:
If ``/etc/protocols`` is missing, than that can be fixed like this https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/eventlet/default.nix#L46-L48 (minus the ``/etc/resolve`` part.

Reply:
unfortunately, the same test then fails with "dns.exception.SyntaxError: service/proto not found"
I would just disable this particular test